flytepropeller/pkg/controller/config/config.go

```diff
@@ -133,6 +134,9 @@ var (
 			MaxSizeInMBForOffloading: 1000, // 1 GB is the default size before failing fast.
 		},
 	}
+
+	// This regex is used to sanitize semver versions passed to IsSupportedSDK checks for literal offloading feature.
```

Comment:
mind including an example of what this matches against with the dev sdk versions?

Reply:
Missed this. Will add in follow up PR.

Reply:
thank you!

Reply:
Added here https://github.com/flyteorg/flyte/pull/5758